brewerdigital-llc/VivoCare
# Task/fa 246

# Description
This PR adds tablet responsiveness to the app. It also refactors the bill pay input and chat/message input.

The app is not perfect on tablets, but it's usable. The proportions are a bit blown up, but to get it perfect, we would've had to design for it and style it separately. The changes here are purely to ensure that all functional pieces of the app are available on the screen, not overlapping each other, or sized somewhat* appropriately.

Look at the `ws` (widthScale), `hs` (heightScale), and `ms` (moderateScale) functions in the style-variables file to get context of the main change. These functions replaced `g.size`.

# Closes issue(s)
FA-246

# Steps to Test
1. Run the app on a real device (phone) to ensure everything's as it should be before testing a tablet
  - It needs to be a real device because the keyboard is a significant factor in these refactors
2. Register a user - Once on the consents page, accept the consent from the PDF modal (you can test both, but I fixed a routing bug here on the pdf modal)
3. Once in the app, ensure all zero states are as they should be.
4. Test messaging and watch the input placement closely.
5. Ensure the book appointment button on the main appointment screen is positioned correctly.
6. Continue testing the rest of the app as a new user.
7. Log out and back in as a demo patient.
8. Test everything from above again but also pay attention to other items (cards, billing functionality, etc) that weren't present on the new user.
9. If you have an iPad, test the app on it. If not, instead of pressing `i` in the terminal to start a simulator, press `shift+i` and you can choose an iPad simulator.
10. Run through steps 2-8 again on the iPad.

# Changes include

# Checklist
